### PR TITLE
[10014] Disable process tests that fail on Windows for py3.7 and 3.8.

### PR DESCRIPTION
**Pull request by adiroiban**
# Why?

twisted.test.test_process.ProcessTests fails on both Azure and GitHub actions VMs on Windows for python 3.7 and 3.8 with iocp and select reactor.

It pass on linux or on Windows with pyhon 3.5 and 3.6

This PR just skips the test to have trunk green again.

I have executed the test on my Windows VM with 3.8 and it pass... so this is related to MS build env VMs.

2 tests were skipped:
* twisted.test.test_process.ProcessTests.test_process
* twisted.test.test_process.ProcessTests.test_manyProcesses

## Contributor Checklist:

* [x] The associated ticket in Trac is here: https://twistedmatrix.com/trac/ticket/10014
* [x] I ran `tox -e black-reformat` to format my patch to meet the [Twisted Coding Standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html)
* [x] I ran additional checks listed at [Getting Your Patch Accepted](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#GettingYourPatchAccepted)
* [x] I have created a newsfragment in src/twisted/newsfragments/ (see: [News files](https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles))
* [x] I have updated the automated tests.
* [x] I have submitted the associated Trac ticket for review by adding the word `review` to the keywords field in Trac, and putting a link to this PR in the comment; it shows up in https://twisted.reviews/ now.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,7 +2,6 @@ Twisted
 =======
 
 |pypi|_
-|travis|_
```
Comment (wsanchez) [inline README.rst:5]: yes

File: tox.ini [hunk 2]
```diff
@@ -32,9 +32,8 @@ minversion=2.4
 skip_missing_interpreters=True
 toxworkdir=build/
 envlist=lint, mypy, black,
-    apidocs,narrativedocs,newsfragment,
+    apidocs, narrativedocs, newsfragment,
     manifest-checker, twine,
-    py36-alldeps-nocov
```
Comment (wsanchez) [inline tox.ini:37]: This removes the only unit test run on the default env list. Is that intentional?

**Review by wsanchez — APPROVED** [review]
I'd prefer to keep some unit tests in the default set, but meh.

File: tox.ini [hunk 3]
```diff
@@ -32,9 +32,8 @@ minversion=2.4
 skip_missing_interpreters=True
 toxworkdir=build/
 envlist=lint, mypy, black,
-    apidocs,narrativedocs,newsfragment,
+    apidocs, narrativedocs, newsfragment,
     manifest-checker, twine,
```
Comment (graingert) [inline tox.ini:36]:
```suggestion
    manifest-checker, twine,
    py38-alldeps-nocov
```

**Review by graingert — APPROVED** [review]

File: tox.ini (hunk 2 above)
Comment (adiroiban) [reply tox.ini:37]: Does the default envlist has any usage? I saw twine in that list...I was thinking that this list is some legacy stuff```

```

**Pull request merged**